resource/docker/container/container.go

```diff
@@ -47,23 +47,23 @@ var (
 type Container struct {
 	*resource.Status
```

Comment:
these and below are still embedding status. Wasn't part of the goal to get away from that?